controllers/controller.js

```diff
@@ -1,25 +1,23 @@
 const db = require("../models");
 
 module.exports = {
+  findById: function(req, res) {
+    db.User.findById(req.params.id)
```

Comment:
Here you/he have/has no 'id: req.params.id' but if that works then sick. Another change to look at